### PR TITLE
oidc override response_type

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: pac4j-oidc/src/test/java/org/pac4j/oidc/run/RunCasOidcWrapper.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Run a manual test for the CAS OpenID Connect wrapper support.
+ *
+ * @author Jerome Leleu
+ * @since 1.9.2
+ */
+public class RunCasOidcWrapper extends RunClient {
```
Comment (leleuj) [inline pac4j-oidc/src/test/java/org/pac4j/oidc/run/RunCasOidcWrapper.java]: @mmoayyed : testing the CAS OIDC server support...

**Comment by leleuj** [comment]
@mmoayyed : Merging for now. I'll re-work on the CAS OIDC server support after: https://github.com/apereo/cas/issues/1945 as been fixed

**Comment by mmoayyed** [comment]
Cool. Are you planning on taking that one as well?

**Comment by leleuj** [comment]
Sure. I'll handle this one.

**Pull request merged**